bin/run

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
```

Comment:
Would it make more sense to replace this with a makefile? `make run`, `make test`, all that?